packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -312,6 +327,7 @@ export default {
         : form.value;
 
       await save({ shippingDetails: shippingAddress });
+      if (isShipmentToBillingSelected.value) await biling.save({ billingDetails: shippingAddress });
```

Comment:
It's funny how simple it is :) 
Could we just keep the formatting consistent with the rest of the file here? Add curly braces around `await billing.save...` (like the line below)?